src/jit/codegencommon.cpp

```diff
@@ -10417,7 +10438,7 @@ bool Compiler::IsMultiRegReturnedType(CORINFO_CLASS_HANDLE hClass)
 }
 #endif // FEATURE_UNIX_AMD64_STRUCT_PASSING
 
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
As i suggested else where we should move #ifdef inside the method with #else returning false on non-hfa targets.  That will avoid the need to enclose all the calls of these methods inside #ifdef FEATURE_HFA